src/main/java/org/fundacionjala/coding/abel/BankOCR.java

```diff
@@ -0,0 +1,84 @@
+package org.fundacionjala.coding.abel;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ *
+ */
+public class BankOCR {
+
+
+    private static final Map<String, String> CONTAINER;
+    public static final int CONT = 3;
+    /**
+     *
+     */
+    public static final int ONCE = 11;
+
+    static {
+        Map<String, String> mapElem = new HashMap<>();
+        mapElem.put("     |  |", "1");
+        mapElem.put(" _  _||_ ", "2");
+        mapElem.put(" _  _| _|", "3");
+        mapElem.put("   |_|  |", "4");
+        mapElem.put(" _ |_  _|", "5");
+        mapElem.put(" _ |_ |_|", "6");
+        mapElem.put(" _   |  |", "7");
+        mapElem.put(" _ |_||_|", "8");
+        mapElem.put(" _ |_| _|", "9");
+        mapElem.put(" _ | ||_|", "0");
+
+        CONTAINER = Collections.unmodifiableMap(mapElem);
+    }
+
+
+    /**
+     * @param string1 string input 1
+     * @param string2 string input  2
+     * @param string3 string input 3
+     * @return value of return
+     */
+    public String scanner(final String string1, final String string2, final String string3) {
+        StringBuilder container = new StringBuilder();
+        StringBuilder newContainer = new StringBuilder();
+        for (int column = 0; column < string1.length(); column += CONT) {
+            container.setLength(0);
+            container.append(string1, column, column + CONT)
+                    .append(string2, column, column + CONT)
+                    .append(string3, column, column + CONT);
+            newContainer.append(CONTAINER.get(container.toString()));
+        }
+        return newContainer.toString().replace("null", "?");
+    }
+
+    /**
+     * @param string1 string input 1
+     * @param string2 string input 2
+     * @param string3 string input 3
+     * @return value of return
+     */
+    public String checksum(final String string1, final String string2, final String string3) {
```

Comment:
This method is doing two things.
Please split them.
one of them only should return a boolean isValidCheckSum